### ISSUE TITLE
DB: Could not find requested WBO global of collection 2 for user 1

### ISSUE THREAD

**Issue by franzhartwig** [owncloud/mozilla_sync#133]
Hi,

I am running OC 8.0.2 on my Raspberry Pi. I use mozilla_sync, syncing Firefox 37.0/Ubuntu and 36/Windows 7.
When syncing, Firefox/Ubuntu causes a log entry:

"app":"mozilla_sync","message":"storageservice.php#124 getWBO():  DB: Could not find requested WBO fxa_credentials of collection 2 for user 1.","level":2,

User 1 is me, collection 2 is "meta" in the table oc_mozilla_sync_collections. Firefox/Win 7 does not cause a log enty, sync setup are the same. services.sync.fxaccounts.enabled ist set to false in about:config. Syncing seems to work without failure. First time seeing this message was 2015-03-06. On 2015-03-05 there was an update from OC 8.0.0-8 => 8.0.0-9, one day later (after the first logging entries) I upgraded 8.0.0-9 => 8.0.1-1. Today, status.php says "version":"8.0.2.0".

What can I do to get rid of the logging entry?

Thank you,

Franz

**Comment by apg1980** [comment]
Me either hit this errors for a few users in my logfile but i cannot see any limitations of functionality with those errors.

**Comment by franzhartwig** [comment]
Thank you for your reply. No, I am not using user_ldap:

$ php console.php app:list
Enabled:
 - bookmarks
 - calendar
 - contacts
 - disablerememberlogin
 - files
 - files_sharing
 - files_trashbin
 - files_versions
 - mozilla_sync
 - user_pwauth

**Comment by alexboss** [comment]
Same problem for me after the upgrade to ownCloud 8.0.2 (deb package) . My version of Mozilla Sync is 1.3.1. Message in the error log file:
```
{"reqId":"8e89304a20c9a44712d080ce0f301b09","remoteAddr":"xxx.xxx.xxx.xxx","app":"mozilla_sync","message":"storageservice.php#124 getWBO():  DB: Could not find requested WBO fxa_credentials of collection 1 for user 1.","level":2,"time":"2015-04-20T16:30:45+00:00","method":"GET","url":"\/remote.php\/mozilla_sync\/1.1\/xxxxxxxxxxxx\/storage\/meta\/fxa_credentials"}
```
User 1 is me and collection 1 is meta. I'm using 1.3.1 because I tried to upgrade the module to 1.4 from the interface, but then permissions are denied and prevent to upgrade module (probably related to the deb package of ownckoud).

**Comment by alexboss** [comment]
After the upgrade to 1.4, the message is still here
>
{"reqId":"590aabb37bfd000573e83e9edff72553","remoteAddr":"xxx.xxx.xxx.xxx","app":"mozilla_sync","message":"storageservice.php#124 getWBO():  DB: Could not find requested WBO fxa_credentials of collection 1 for user 1.","level":2,"time":"2015-04-20T17:00:59+00:00","method":"GET","url":"\/remote.php\/mozilla_sync\/1.1\/xxxxxxxxxxxx\/storage\/meta\/fxa_credentials"}
>

**Comment by franzhartwig** [comment]
I am using Mozilla Sync 1.4, installed together with Owncloud - using the repo from Opensuse.
Still Firefox (now version 38) on Ubuntu causes the message, Firefox (36) in Win 7 did not. Firefox 37.0.1 (just updated) does. Seems there is something changed with Firefox 37 an newer?

**Comment by hannesa2** [comment]
FF 38.0.5 on Mac 10.10 has the same issue

**Comment by ffflorian** [comment]
FF 39.0 with ownCloud 8.1 and mozilla_sync 1.4 still has this problem.

**Comment by RealRancor** [comment]
Just ignore the error as the sync should still work. It also probably don't worth to fix this as the old sync used by this app is kicked out of firefox in a few months.

**Comment by ffflorian** [comment]
@RealRancor yes the sync still works so I'm just going to ignore it. Are there some plans regarding ownCloud and the new sync already?

**Comment by RealRancor** [comment]
No idea, sorry. An open discussion of the sync is available here: https://github.com/owncloud/mozilla_sync/issues/33

**Comment by godfuture** [comment]
I am trying to pair my android device with firefox sync on owncloud, but without luck.
The old sync vanished in ff and palemoon does not seem to work as well. Palemoon sync assist says:
> Incorrect account name or password.

As both is correct (checked a hundred times), I guess this error from mozilla_sync app seems valid.
